.gitattributes

```diff
@@ -0,0 +1,2 @@
+**/*.snap linguist-generated=true
```

Comment:
This is a QoL improvement for the snapshots; it's what tells Github to do below "Some generated files are not rendered by default."

[More details here.](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github) Happy to undo this.